tools/makecss.py

```diff
@@ -12,11 +12,11 @@ def genCSS(font, base):
         style = "oblique"
     weight = font["OS/2"].usWeightClass
     family = font["name"].getName(nameID=1, platformID=3, platEncID=1)
-    family = str(family).replace(" ", "")
+    family = str(family).replace(" ", "").replace("\0", "")
 
     css = """
 @font-face {
-    font-family: %(family)sWeb;
+    font-family: %(family)sArabicMathJax;
```

Comment:
Please revert the change, there's no need for making this change here.

Users (i.e. UI developers, including the font) usually do that on their own.

```suggestion
    font-family: %(family)sWeb;
```

Reply:
Done @OmarIthawi